game/views/game_view.py

```diff
@@ -26,9 +26,13 @@ class GameState(Enum):
 
 class GameView(arcade.View):
     door_open_sound = arcade.Sound("game/assets/sound_effects/door_open.wav")
+    level_list = ["level1", "level2"]
```

Comment:
why not just use the int counter we already have and just do something like `f"level{self.level}"`

Reply:
sure if you want each level to have that strict naming scheme